ModelBundle/Migrations/MongoDB/Version20160601155518.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Migrations\MongoDB;
+
+use AntiMattr\MongoDB\Migrations\AbstractMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Doctrine\MongoDB\Database;
+use Symfony\Component\Yaml\Parser as YamlParser;
+use OpenOrchestra\ModelInterface\Repository\RepositoryTrait\KeywordableTraitInterface;
+
+class Version20160601155518 extends AbstractMigration implements ContainerAwareInterface
+{
+    private $container;
+    protected $collections;
+    protected $configuration;
+
+    /**
+     * Set the container
+     *
+     * @param ContainerInterface $container
+     */
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+        $this->getKeywordableCollection();
+        $this->loadConfiguration();
+    }
+
+    /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return "change stored keyword to dbref";
+    }
+
+    /**
+     * @param Database $db
+     */
+    public function up(Database $db)
+    {
+        $databaseName = $db->getName();
+
+        foreach ($this->collections as $collection) {
+            $db->execute('
+                db.' . $collection . '.find().forEach(function(item) {
+                    if (typeof item.keywords != \'undefined\') {
```

Comment:
@itkg-nanne : I find this very greedy and dangerous. It should only search where we want (ie only several determined collections and only determined attributes) to be lighter + to avoid a bad transformation on a custom collection with an attribute named keywords that has nothing to deal with the migration